kaska/kaska.py

```diff
@@ -110,54 +115,60 @@ def _run_smoother(self, dates, parameter_block):
         lai = -2 * np.log(parameter_block[-2, :, :, :])
         cab = -100*np.log(parameter_block[1, :, :, :])
         cbrown = parameter_block[2, :, :, :]
+        if self.save_sgl_inversion:
+            save_output_parameters(dates, self.observations, 
+                self.output_folder/"single_imgs/", ["lai", "cab", "cbrown"],
```

Comment:
```suggestion
                self.output_folder / "single_imgs" /, ["lai", "cab", "cbrown"],
```
If we using `pathlib` then it's nicer to get the `/` outside the strings.

Reply:
Done